### PR TITLE
add logs monitor API documentation

### PR DESCRIPTION
**Pull request by PierreVanRoy**
### What does this PR do?

Document how to create logs monitor with the API

### Motivation
https://trello.com/c/hKRkb0Of/2215-log-monitor-api-logs-sample-not-included-in-alert-from-logs-monitor-created-through-api

### Preview link

https://docs-staging.datadoghq.com/pvr/log-monitor-api/api/?lang=bash#create-a-monitor

### CODE REVIEW

File: content/en/api/monitors/monitors_create.md [hunk 1]
```diff
@@ -89,6 +89,18 @@ If you manage and deploy monitors programmatically, it's easier to define the mo
...
+
+    `logs(query).index(index_name).rollup(rollup_method[, measure]).last(time_window) operator #`
+    
+    *   **`query`** The search query - following the [Log search syntax][6] .
+    *   **`index_name`** For multi-index organizations, the log index in which the request is performed.
+    *   **`rollup_method`** the stats rollup method. count, avg and cardinality are the only supported method now
+    *   **`measure`** For avg and cardinality rollup_method, specify the measure or the facet name you want to use.
+    *   **`time_window`** #m (5, 10, 15, or 30), #h (1, 2, or 4, 24)
+    *   **`operator`** <, <=, >, >=, ==, or !=
+    *   **`#`** an integer or decimal number used to set the threshold
```
Comment (apigirl) [inline content/en/api/monitors/monitors_create.md]:
```suggestion
    *   **`#`** an integer or decimal number used to set the threshold.
```

File: content/en/api/monitors/monitors_create.md [hunk 2]
```diff
@@ -89,6 +89,18 @@ If you manage and deploy monitors programmatically, it's easier to define the mo
...
+    ##### Logs Alert Query
+
+    `logs(query).index(index_name).rollup(rollup_method[, measure]).last(time_window) operator #`
+    
+    *   **`query`** The search query - following the [Log search syntax][6] .
+    *   **`index_name`** For multi-index organizations, the log index in which the request is performed.
+    *   **`rollup_method`** the stats rollup method. count, avg and cardinality are the only supported method now
+    *   **`measure`** For avg and cardinality rollup_method, specify the measure or the facet name you want to use.
+    *   **`time_window`** #m (5, 10, 15, or 30), #h (1, 2, or 4, 24)
+    *   **`operator`** <, <=, >, >=, ==, or !=
```
Comment (apigirl) [inline content/en/api/monitors/monitors_create.md]:
```suggestion
    *   **`operator`** `<`, `<=`, `>`, `>=`, `==`, or `!=`.
```

File: content/en/api/monitors/monitors_create.md [hunk 3]
```diff
@@ -160,6 +172,22 @@ If you manage and deploy monitors programmatically, it's easier to define the mo
...
+    Example: `{'ok': 1, 'critical': 1, 'warning': 1}`
+    
+    - **`aggregation`** a dictionary of `type`, `metric` and `groupeBy`:
+        - `type`  3 types are supported: `count`, `cardinality` and `avg`
+        - `metric`:  for `cardinality` name of the facet. For `avg` name of the metric. for `count`just put `count` as metric  
+        - `groupeBy` name of the facet on which you want to group by.
+    
+    Example: `{"metric": "count","type": "count","groupBy": "core_service"}`
+    
+    - **`enable_logs_sample`** a Boolean to add samples or values to the notification message. Default: True
```
Comment (apigirl) [inline content/en/api/monitors/monitors_create.md]:
```suggestion
    - **`enable_logs_sample`** a Boolean to add samples or values to the notification message. Default: `True`
```

**Comment by PierreVanRoy** [comment]
All those suggestions are great!

**Pull request merged**